packages/xml-views-validation/api.d.ts

```diff
@@ -1,22 +1,31 @@
-import { XMLDocument } from "@xml-tools/ast";
+import { XMLDocument, XMLElement, XMLAttribute } from "@xml-tools/ast";
 import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
 import { OffsetRange } from "@ui5-language-assistant/logic-utils";
+import { UI5Validators } from "./src/validate-xml-views";
 
 export function validateXMLView(opts: {
+  validators: UI5Validators;
```

Comment:
maybe call this type: `ValidatorsConfig`?

Reply:
UI5ValidatorsConfig?